committees/migrations/0009_merge_20230121_1237.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 3.2.16 on 2023-01-21 11:37
```

Comment:
This migration is empty?

Reply:
It was a merge migrations that merges two branches, I reordered some migration orders so this migration is no longer needed.